tests/test_benchmark/expected-results/benchmark.tsv

```diff
@@ -0,0 +1,6 @@
+jobid	rule	wildcards	resources	input_size_mb	threads	s	h:m:s	max_rss	max_vms	max_uss	max_pss	io_in	io_out	mean_load	cpu_time	input_file_size_mb
+2	bench_script	NA	_cores=1;_nodes=1;tmpdir=/tmp	0.0	1	1.1346	0:00:01	32.8	45.8	19.93	23.09	0.0	0.0	0.0	0.09	NA
```

Comment:
I would maybe move the potentially very long columns to the right hand side:
- resources
- input_size_mb

The most relevant ones could be moved further left:
- CPU
- wallclock time

Reply:
I've moved resources and input_size_mb to right hand side, jobid, rule, and wildcards stay on the left hand side since they serve as "index". Other columnd remain in the same order as original benchmark output